touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -326,9 +329,11 @@ WebFormSession.prototype.submitForm = function(form) {
 
     accumulate_answers = function(o) {
         if (ko.utils.unwrapObservable(o.type) !== 'question') {
-            $.each(o.children(), function(i, val) {
-                accumulate_answers(val);
-            });
+            if(typeof o.children == 'function') {
```

Comment:
the way to do this in knockout is to use `ko.utils.unwrapObservable` which will unwrap the value if it's an observable (a function) or if it's not, it will just return the plain value